src/runtime/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ if(LLVM)
   endif()
   FILE(READ "${CMAKE_CURRENT_SOURCE_DIR}/../include/lean/lean.h" LEAN_H)
   # generate LLVM IR for `static inline` definitions in lean.h for the LLVM backend
-  string(REPLACE "static inline" "__attribute__((always_inline))" LEAN_H "${LEAN_H}")
+  string(REPLACE "static inline" "__attribute__((always_inline, weak))" LEAN_H "${LEAN_H}")
```

Comment:
Wrong PR?

Reply:
I'm unsure where this change should live. 

I was wondering if it ought to be here, since it sets up the outlining for the backend, or in the "main PR" which sets up the build system.

I'll move it over to the other PR, since I guess that you feel it's more appropriate there.